src/comp/FollowButton.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import { Button } from "@material-ui/core";
+import * as styles from "css/comp/FollowButton.module.css";
+
+type userDetailsProps = {
+  isFollow: boolean;
+  updateFollowingList: () => void;
+};
+
+const FollowButton = (props: userDetailsProps) => {
+  const buttonContents = (isFollow: boolean) => {
+    let followButtonText = "Follow";
+    if (isFollow) {
+      followButtonText = "Unfollow";
+    }
+    return <p>{followButtonText}</p>;
```

Comment:
細かいんですが、変数に入れなくても良くないですか
```suggestion
    if (isFollow) {
      return <p>Unfollow</p>
    }
    return <p>Follow</p>;
```